lib/gitarro/opt_parser.rb

```diff
@@ -73,6 +73,14 @@ def https_opt(opt)
     opt.on('--https', https_desc) { |https| @options[:https] = https }
   end
 
+  def cachehttp_opt(opt)
+    desc = 'Custom path where http cache for gitarro is stored' \
+           'by default is set to /tmp/gitarro/httpcache'
+    opt.on('-k', "--cachepath 'CACHEPATH'", desc) do |cache|
+      @options[:cachehttp] = cache
```

Comment:
Why not using ```cachehttp```instead of of ```cache``` as we do for the other options?

Reply:
cache there is just a throw-away variable, and using cachehttp maybe can make confusion with the real variable

Reply:
but ok, is cosmetical i will add `cache_http`